test/functional/TestUtilities/src/org/openj9/test/attachAPI/TargetVM.java

```diff
@@ -61,14 +63,18 @@ public static void main(String[] args) {
 				return;
 			}
 			Properties p = new Properties();
-			p.setProperty("j9vm.test.attach.testproperty2", "5678def");
+			p.setProperty(propertyKey, propertyValue);
 			String cmd = "";
 			cmd = waitForTermination(inRdr, pid, cmd);
 			if (null == cmd) {
 				System.out.println(pid + " stdin closed unexpectedly");
 			} else {
 				System.out.println(pid + " terminated by \"" + cmd + "\"");
 			}
+			/* force a reference to keep the Properties object alive */
+			if (!p.getProperty(propertyKey).equals(propertyValue)) {
```

Comment:
The use of `equals` should be inverted to catch the case where the property is gone.

Reply:
Right.  Theoretically the property should not disappear (the code is purely to keep the object alive), but your suggestion is better style.